packages/yew-macro/src/stringify.rs

```diff
@@ -75,26 +78,31 @@ impl Stringify for LitStr {
         }
     }
 }
+
 impl Stringify for Lit {
     fn try_into_lit(&self) -> Option<LitStr> {
-        let s = match self {
+        let mut buf = [0; size_of::<char>()];
+        let s: Cow<'_, str> = match self {
             Lit::Str(v) => return v.try_into_lit(),
-            Lit::Char(v) => v.value().to_string(),
-            Lit::Int(v) => v.base10_digits().to_string(),
-            Lit::Float(v) => v.base10_digits().to_string(),
-            Lit::Bool(_) | Lit::ByteStr(_) | Lit::Byte(_) | Lit::Verbatim(_) => return None,
-            _ => unreachable!("unknown Lit"),
+            Lit::Char(v) => (&*v.value().encode_utf8(&mut buf)).into(),
```

Comment:
```suggestion
            Lit::Char(v) => (&*v.value().encode_utf8(&mut [0; size_of::<char>()])).into(),
```

Reply:
This doesn't pass borrow checking, "temporary value dropped while borrowed"